src/board/boards/blcu/blcu.go

```diff
@@ -0,0 +1,128 @@
+package blcu
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"strconv"
+
+	excel_models "github.com/HyperloopUPV-H8/Backend-H8/excel_adapter/models"
+	"github.com/HyperloopUPV-H8/Backend-H8/vehicle/models"
+	"github.com/rs/zerolog"
+	trace "github.com/rs/zerolog/log"
+)
+
+// TODO: Get these values from the TOML
+const (
+	BLCU_COMPONENT_NAME       = "blcu"
+	BLCU_BOARD_NAME           = "blcu"
+	BLCU_HANDLER_NAME         = "blcu"
+	BLCU_UPLOAD_ORDER_ID      = 700
+	BLCU_DOWNLOAD_ORDER_ID    = 701
+	BLCU_UPLOAD_ORDER_FIELD   = "board"
+	BLCU_DOWNLOAD_ORDER_FIELD = "board"
+	BLCU_INPUT_CHAN_BUF       = 100
+	BLCU_ACK_CHAN_BUF         = 1
+	BLCU_ACK_PACKET_NAME      = "tftp_ack"
+)
+
+type BLCU struct {
+	addr  string
+	ackID uint16
+
+	inputChannel chan models.Update
+	ackChannel   chan struct{}
+
+	sendOrder   func(models.Order) error
+	sendMessage func(topic string, payload any, targets ...string) error
+
+	trace zerolog.Logger
+}
+
+func NewBLCU() *BLCU {
```

Comment:
Why do you return a poiter?

Reply:
It is necessary because the interface methods require a pointer